src/matrixHelpers/formioScript/index.prod.js

```diff
@@ -0,0 +1,42 @@
+import { createScripts } from "./createScripts";
+
+const scripts = [
+  {
+    type: "script",
+    src: "https://static.qgov.net.au/formio-qld/v1/v1.x.x-latest/formio.full.min.js",
```

Comment:
v1/v1.x.x-latest may need to be placed into a variable so it is changed all at once (central to all environments)

Reply:
there may be some glitches if 2 forms in the same page with different version/env settings, with current solution, the settings of the later form will override the former one